doc-content/enterprise-only/admin-and-config/managing-business-central-database-connection-con.adoc

```diff
@@ -0,0 +1,23 @@
+[id='managing-business-central-database-connection-con']
+= Database connection
+
+{CENTRAL} provides a dedicated `UserGroupCallback` implementation for database server with {PRODUCT} to enable the user task service to retrieve information on users and groups (roles) directly from databases.
```

Comment:
Suggested rewrite: 
Business Central provides a dedicated UserGroupCallback implementation for the database server with Red Hat Process Automation Manager to enable the user task service. The user task service helps in retrieving information on users and groups (roles) directly from databases.
 //instead of a long sentence, we can split it in two sentences
wdyt?

Reply:
Done